### PR TITLE
NF: use enum for direction

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
This was detected as a problem with switch and an empty case. But it seems this was more important as far as typing goes

### CODE REVIEW

**Comment by mikehardy** [comment]
took 2 conflicts from previous merges

**Comment by Arthur-Milchior** [comment]
Resolved

File: AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/NoteService.java [hunk 1]
```diff
@@ -36,6 +36,7 @@
 import java.io.File;
 import java.io.IOException;
 
+@SuppressWarnings("ALL")
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/NoteService.java:39]: Why?

**Review by david-allison-1 — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/NoteService.java (hunk 1 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/NoteService.java:39]: Seems an accident with Linter suppression of warning.

**Review by david-allison-1 — APPROVED** [review]
👍

**Comment by mikehardy** [comment]
@Arthur-Milchior took another conflict :-), but, we're almost through merging all these so maybe it will quiet down

**Review by mikehardy — APPROVED** [review]

**Pull request merged**